packages/common/src/utils/formatters/formatters.ts

```diff
@@ -6,9 +6,6 @@ export const Formatter = {
   // TODO: Do something better than naively rounding to 2 decimal places
   tax: (tax: number, withParens = true) =>
     `${withParens ? '(' : ''}${(tax ?? 0).toFixed(2)}%${withParens ? ')' : ''}`,
-  date: (date: Date): string =>
-    `${date.getDate()}/${date.getMonth() + 1}/${date.getFullYear()}`,
-  localisedDate: (date: Date): string => format(date, 'P'),
```

Comment:
Never used, can't see a reason for it.

Reply:
I think a remnant from a time when the api was returning dates, or being parsed on fetch